README.md

```diff
@@ -4,34 +4,72 @@ This repository contains the processing pipeline for IceFloeTracker.jl and ancil
 
 ## SOIT Integration
 
-The [Satellite Overpass Identification Tool](https://zenodo.org/record/6475619#.ZBhat-zMJUe) is called to generate a list of satellite times for both Aqua and Terra in the area of interest. This program is written in Python and its dependencies are pulled from a docker container at `docker://brownccv/icefloetracker-fetchdata:latest`.
+The [Satellite Overpass Identification Tool](https://zenodo.org/record/6475619#.ZBhat-zMJUe) is called to generate a list of satellite times for both Aqua and Terra in the area of interest. This program is written in Python and its dependencies are pulled from a docker container at `docker://brownccv/icefloetracker-fetchdata:main`.  
 
-**Note:** the `pass_time_cylc.py` script in this project can be adapted to include additional satellites available in the [space-track.org](https://www.space-track.org/) repository.
+**Note:** The `pass_time_cylc.py` script in this project can be adapted to include additional satellites available in the [space-track.org](https://www.space-track.org/) repository.
 
-Export SOIT username/password to environment variable:
+## Cylc to run the pipeline
+
+Cylc is used to encode the entire pipeline from start to finish and relies on the command line scripts to automate the workflow. The `config/cylc_hpc/flow.cylc` file should be suitable for runs on HPC systems. To run Cylc locally, the `config/cylc_local_flow.cylc` file is used.
+
+### Running the workflow locally
+
+When running locally, make sure Docker Desktop client is running. (https://www.docker.com/products/docker-desktop/)
+
+1. Build a virtual environment and install Cylc
+   - [ ] `cd <your-project-path>/ice-floe-tracker-pipeline`
+   - [ ] `conda env create -f ./config/ift-env.yaml`
+   - [ ] `conda activate ift-env`
+
+**Note:** Depending on your existing Conda config, you may need to update your `.condarc` file to: `auto_activate_base: false` if you get errors running your first Cylc workflow.
+
+2. Register an account with [space-track.org](https://www.space-track.org/) for SOIT
+
+3. Export SOIT username/password to environment variable
    - [ ] From your home directory`nano .bash_profile`
    - [ ] add `export HISTCONTROL=ignoreboth` to the bottom of your .bash_profile
         * this will ensure that your username/password are not stored in history
-        * when exporting the following environment variables, there must a space in front of each command
+        * when exporting the following environment variables, there __must__ be a space in front of each command
    - [ ] ` export SPACEUSER=<firstname>_<lastname>@brown.edu`
    - [ ] ` export SPACEPSWD=<password>`
 
-## Cylc to run the pipeline
+4. Install your workflow, run it, and monitor with the text-based user interface (tui)
```

Comment:
tui or TUI?

Reply:
Good catch. The Cylc docs seem to use tui, Tui, and TUI. We'll go with TUI!